Sources/Script/Script.swift

```diff
@@ -75,8 +75,8 @@ public class Script {
 
             var macOS: String {
                 let version = ProcessInfo.processInfo.operatingSystemVersion
-                if version.majorVersion == 11 && version.minorVersion == 0 {
-                    // swift-tools-version 5.1 doesn’t have the v11 value
+                if version.majorVersion == 10 && version.minorVersion == 16 {
+                    // SwiftPM sucks both v10_16 and v11 trigger errors (lol)
```

Comment:
now it's  v11

'v10_16' has been renamed to 'v11'

Reply:
But the `swift-tools-version` must be set to `5.3`. Additionally, on macOS 11.1, we should still use `v11`, not `v11_1`.

Reply:
@Frizlab should all `version.majorVersion == 11 && version.minorVersion >= 1` map to `v11` for now? Or do you think it'd be better to handle `version.majorVersion == 11 && version.minorVersion == 1` as an exception? 

I'm asking because I have a similar PR in progress passing all tests, but I'm not handling this case currently: https://github.com/mxcl/swift-sh/pull/134

Reply:
I don't really know; following my gut I'd link to "v\(major)", simply. But I have no data to back this.

Reply:
So, trying this: https://github.com/mxcl/swift-sh/commit/e5ef3432f8e2df9f950ad63ba3407d6d2aa9a8a7